lib/cext/ruby.h

```diff
@@ -78,7 +78,7 @@ extern "C" {
 
 // Basic types
 
-typedef void *VALUE;
+typedef void* VALUE;
 typedef long SIGNED_VALUE;
```

Comment:
That one is horribly confusing, do you really want it?
It's like if we alias `void` and `*VALUE`.